track_local_static.go

```diff
@@ -327,10 +350,57 @@ func (s *TrackLocalStaticSample) WriteInterleavedSample(sample media.Sample, onR
 	return util.FlattenErrs(writeErrs)
 }
 
+func addExtensions(sample media.Sample, packets []*rtp.Packet) error {
+	var sampleAttr byte = 0
+	position, err := getExtensionVal("HYPERSCALE_RTP_EXTENSION_FIRST_PACKET_ATTR_POS")
```

Comment:
general question - how come position and err are with := is it because they are in an if?

Reply:
yes. the if creates a new scope that shadows the function scope.